architecture/additional_concepts/service_catalog.adoc

```diff
@@ -0,0 +1,321 @@
+= Service Catalog
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[service-catalog-overview]]
+== Overview
+
+[NOTE]
+====
+The service catalog is currently a Technology Preview feature.
+ifdef::openshift-origin,openshift-enterprise[]
+To opt-in during installation or upgrade, see Installation & Configuration (TODO link).
+endif::[]
+====
+
+When developing microservices-based applications to run on cloud native
+platforms, there can be many ways to provision different resources and share
+their coordinates (URL),  credentials, and configuration, depending on the
+service provider and the platform.
+
+To give developers a more seamless experience, {product-title} includes a
+_service catalog_, an implementation of the
+link:https://openservicebrokerapi.org/[Open Service Broker API] (OSB API) for
+Kubernetes. This allows users to connect any of their applications deployed in
+{product-title} to a wide variety of service brokers in a microservices-based
+environment.
+
+The service catalog allows cluster administrators to integrate with multiple
+platforms using a single API specification. The {product-title} web console
+aggregates and lists the contents of any configured service brokers, allowing
+users to discover and instantiate those services for use with their
+applications.
+
+As a result, service users benefit from ease and consistency of use across
+different types of services from different providers, while service providers
+benefit from having one integration point that gives them access to multiple
+platforms.
+
+[[service-catalog-design]]
+== Design
+
+The design of the service catalog follows this basic workflow:
+
+. A cluster administrator registers a _service broker_ with their {product-title} cluster.
+. The service broker specifies the set of _service classes_ and variations of
+those services (_service plans_) to {product-title} that should be made
+available to users.
+. Users discover the services that are available for use. For example, a database
+as a service called BestDataBase.
+. A user requests a new _instance_ of a service. For example, a BestDataBase
+instance named `my_db`.
+. A user links, or _binds_, a service instance to a set of pods (their
+application). For example, the `my_db` service instance is bound to the user's
+application called `my_app`.
+
+This infrastructure allows for a loose coupling between applications running in
+{product-title} and the services they use. This allows for the application that
+uses those services to focus on its own business logic while leaving the
+management of these services to the provider.
+
+image::svc-catalog-arch.png["Service Catalog Architecture"]
+
+[[service-catalog-concepts-terminology]]
+== Concepts and Terminology
+
+The following concepts and resources are the building blocks of the service
+catalog:
+
+Application::
+{product-title} has a core concept of
```

Comment:
I think this paragraph is round the wrong way - I'd start with what an application is.